scripts/train.py

```diff
@@ -1,34 +1,32 @@
 import os
 import click
 import pickle
+
+import sys
 from sklearn.neural_network import MLPClassifier
 from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
 from pkspace.utils import trainer
 
 
 @click.command()
-@click.option('--PKSpace', 'dataset_mode', flag_value='PKSpace',
-              default=True)
-@click.option('--PKLot', 'dataset_mode', flag_value='PKLot')
-@click.option('--MLP', 'method', flag_value='MLP',
-              default=True, help='Method to be used for prediction')
-@click.option('--dataset_dir', required=True,
-              help='Directory of dataset for model to be trained on')
-@click.option('--output', default=None,
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.option('--model_type', '-mt', type=click.Choice(['MLP']), default='MLP',
+              help='Type of model to be trained')
+@click.argument('dataset_dir')
```

Comment:
@rstevanak here we can probably add `type=click.Path(exists=True)` so click would automatically handle the existence of the directory so we do not have to check it.